src/Utils.js

```diff
@@ -127,6 +132,14 @@ const Utils = {
 
   getMonthsDiff(d1, d2) {
     return d2.getMonth() - d1.getMonth() + (12 * (d2.getFullYear() - d1.getFullYear()));
+  },
+
+  getNextMonthLabel() {
+    return LABELS.NEXT;
+  },
+
+  getPreviousMonthLabel() {
```

Comment:
We could change the function names to `getPreviousLabel` and `getNextLabel` then...